### PR TITLE
Feature/todos crud frontend

### PR DESCRIPTION
**Pull request by OlekasndrG**

### CODE REVIEW

File: packages/frontend/src/modules/Todos/TodoItem/TodoItem.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+							type="button"
+						/>
+						<Link to={ROUTER_KEYS.DASHBOARD} className={TodoLink}>
+							Back
+						</Link>
+						<Button
+							text={'Delete'}
+							onClick={deleteTodoAndGoToList}
+							type="button"
+						/>
```
Comment (malyshkoserhii) [inline packages/frontend/src/modules/Todos/TodoItem/TodoItem.tsx:60]: 1. code duplication. Create reusable component

File: packages/frontend/src/modules/Todos/TodoItem/TodoItem.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+							to={`${ROUTER_KEYS.DASHBOARD}/${todo.id}`}
+							className={TodoLink}
+						>
+							View
+						</Link>
+						<Button
+							text={'Delete'}
+							onClick={deleteTodo}
+							type="button"
+						/>
```
Comment (malyshkoserhii) [inline packages/frontend/src/modules/Todos/TodoItem/TodoItem.tsx:74]: 1. code duplication. Create reusable component

File: packages/frontend/src/modules/Todos/TodoPage/TodoPage.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	};
+	const openUpdateModal = (): void => {
+		setOpenaModal(true);
+	};
+
+	const updateTodoFunc = (todo: UpdateTodoType): void => {
+		updateTodo(+id, todo);
+		closeUpdateModal();
+	};
+	const initState = todo
```
Comment (malyshkoserhii) [inline packages/frontend/src/modules/Todos/TodoPage/TodoPage.tsx:37]: useMemo

File: packages/frontend/src/modules/Todos/Tododashboard/TodoDesktopDashboard/TodoDesktopDashboard.tsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+										className={TodoLink}
+									>
+										View
+									</Link>
+
+									<Button
+										text={'Delete'}
+										onClick={() => removeTodo(todo.id)}
+										type="button"
+									/>
```
Comment (malyshkoserhii) [inline packages/frontend/src/modules/Todos/Tododashboard/TodoDesktopDashboard/TodoDesktopDashboard.tsx:55]: 1. code duplication. Create reusable component

Comment (OlekasndrG) [reply packages/frontend/src/modules/Todos/Tododashboard/TodoDesktopDashboard/TodoDesktopDashboard.tsx]: Done

File: packages/frontend/src/modules/Todos/TodoPage/TodoPage.tsx (hunk 3 above)
Comment (OlekasndrG) [reply packages/frontend/src/modules/Todos/TodoPage/TodoPage.tsx]: Done

File: packages/frontend/src/modules/Todos/TodoItem/TodoItem.tsx (hunk 2 above)
Comment (OlekasndrG) [reply packages/frontend/src/modules/Todos/TodoItem/TodoItem.tsx]: Done

File: packages/frontend/src/modules/Todos/TodoItem/TodoItem.tsx (hunk 1 above)
Comment (OlekasndrG) [reply packages/frontend/src/modules/Todos/TodoItem/TodoItem.tsx:59]: Done

**Pull request merged**